src/injected/focus-change-handler.ts

```diff
@@ -34,4 +36,29 @@ export class FocusChangeHandler {
         this.scrollingController.processRequest(scrollingMessage);
         this.targetPageActionMessageCreator.scrollRequested();
     };
+
+    private getTarget(storeData: TargetPageStoreData): string[] {
+        return (
+            storeData.visualizationStoreData.focusedTarget || this.getCardResultTarget(storeData)
```

Comment:
would it be worth it to have a comment here to explain why we need to check `visualizationStoreData` and not only the `cardSelectionStoreData`?